e2e/core/src/pl/mk5/gdx/fireapp/e2e/GdxFireappE2ETests.java

```diff
@@ -55,8 +56,8 @@ public GdxFireappE2ETests() {
         e2ETestRunner.addNext(new AuthAnonymousTest());
         e2ETestRunner.addNext(new AuthCreateUserEmailPasswordTest());
         e2ETestRunner.addNext(new AuthSignInUserEmailPasswordTest());
-//        e2ETestRunner.addNext(AuthGoogleSignInTest.class);
-        e2ETestRunner.addNext(new AuthAppleSignInTest(), 360);
+//        e2ETestRunner.addNext(new AuthGoogleSignInTest(), 360);
```

Comment:
Why are these e2e tests failing ?

Reply:
they are not failing / but they required user interaction. So they cannot be run from ci. They need to be run manually